pkg/helm/repo.go

```diff
@@ -83,8 +83,9 @@ func allProviders(settings *cli.EnvSettings, httpRequestTimeout time.Duration) g
 // some difference with `helm repo add` command
 // 1. when the same repo name is added, it will overwrite
 // 2. many options we do not need now are not supported yet.
-func RepoAdd(ctx context.Context, logger logr.Logger, name, url, username, password string, httpRequestTimeout time.Duration) (err error) {
-	entry := repo.Entry{Name: name, URL: url, Username: username, Password: password}
+func RepoAdd(ctx context.Context, logger logr.Logger, name, url string, entry repo.Entry, httpRequestTimeout time.Duration) (err error) {
```

Comment:
what about just use `entry` as args, drop `name` and `url`?

Reply:
Yes, just assigning values ​​​​before passing parameters, it doesn’t feel like it has much impact